app/models/namespace.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# The namespace is used to group repositories by the owner and to allow setting
+# repository-local permissions for teams and users.
+class Namespace < Sequel::Model
+  plugin :timestamps
+
+  many_to_one :organizational_unit
```

Comment:
Shouldn't this be `one_to_one`? I think we should separate the namespace that belongs to a user/team and the ones a user/team has access to. If we separate it, a user/team can only ever have one namespace that belongs to it.